File: SoftUni Lessons/Python Development/Python Basics April 2019/Lessons and Problems/05 - Conditional Statements Exercise/03. Speed Info .py
speed = float(input())

if speed <= 10:
    print("slow")
elif speed > 10 and speed <= 50:
    print("average")
elif speed > 50 and speed <= 150:
    print("fast")
elif speed > 150 and speed <= 1000:
    print("ultra fast")
else:
    print("extremely fast")

File: SoftUni Lessons/JavaScript Development/JavaScript Basics March 2020/08. УПРАЖНЕНИЕ ПРОВЕРКИ/03. Speed Info.js
function speedInfo(input) {
    let speed = Number(input.shift());
    
    if (speed <= 10) {
        console.log("slow");
    } else if (speed <= 50) {
        console.log("average");
    } else if (speed <= 150) {
        console.log("fast");
    } else if (speed <= 1000) {
        console.log("ultra fast");
    } else if (speed > 1000) {
        console.log("extremely fast");
    }
}

speedInfo([8]); // Expected Output: slow
speedInfo([49.5]); // Expected Output: average
speedInfo([126]); // Expected Output: fast
speedInfo([160]); // Expected Output: ultra fast
speedInfo([3500]); // Expected Output: extremely fast
